### DISCUSSION TITLE
How to hook onto a SIWE signout event?

### DISCUSSION THREAD

**Discussion by montycheese** [family/connectkit#55] [Q&A]
I have a react project set up with an express backend that uses SIWE to manage signin/credentials.

With the current ConnectKit SIWE integration, users can signout through the connect kit modal.

However, our application needs to zero out several app contexts and app level state upon a signout event. 

Is there a native way to hook into a signout event from SIWE to allow achieving this goal?

**Comment by lochie** [comment]
Hey @montycheese, there's [an issue reported](https://github.com/family/connectkit/issues/51) for this that we're actively looking into.
We've also got [a PR started](https://github.com/family/connectkit/pull/69) with some additional SIWE updates.

Hoping to have this updated this week to allow easy use of the `signOut` call 🙂
